src/config.ts

```diff
@@ -821,8 +821,9 @@ function assertEnvVarType(name: string, value: any, expectedType: EnvVarType): a
             return parseUtils.parseJsonStringForMetaTransactionRateLimitConfigOrThrow(value);
         case EnvVarType.APIKeys:
             assert.isString(name, value);
-            const apiKeys = (value as string).split(',');
+            const apiKeys = (value as string).split(',').filter((key) => !!key.trim());
             apiKeys.forEach((apiKey) => {
+                if (!apiKey) return;
```

Comment:
This line can be removed when you filter out the empty keys

Reply:
done!